crates/proof-of-sql/src/proof_primitive/dory/rand_util.rs

```diff
@@ -16,6 +16,7 @@ pub fn test_seed_rng(seed: [u8; 32]) -> impl Rng {
     StdRng::from_seed(seed)
 }
 
+#[allow(dead_code)]
```

Comment:
Im not sure how to fix this, there is a web of features and re exports that Im not really familiar with here. All other checks succeed, clippy just complains about this one spot:

```bash
cargo clippy --all-targets --all-features -- -D warnings
```

```
error: unused import: `rand_util::rand_G_vecs`
  --> crates/proof-of-sql/src/proof_primitive/dory/mod.rs:29:5
   |
29 | use rand_util::rand_G_vecs;
   |     ^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D unused-imports` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unused_imports)]`

error: function `rand_G_vecs` is never used
  --> crates/proof-of-sql/src/proof_primitive/dory/rand_util.rs:20:8
   |
20 | pub fn rand_G_vecs<R>(nu: usize, rng: &mut R) -> (Vec<G1Affine>, Vec<G2Affine>)
   |        ^^^^^^^^^^^
   |
   = note: `-D dead-code` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(dead_code)]`

error: could not compile `proof-of-sql` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```